src/components/text-editor/examples/text-editor-custom-triggers.tsx

```diff
@@ -163,9 +127,7 @@ export class TextEditorCustomTriggersExample {
         }
 
         if (event.key === ESCAPE) {
-            this.close.emit();
             this.isPickerOpen = false;
-            // @TODO: close the trigger session
```

Comment:
What do we mean with this TODO?

Reply:
I've fixed it. We should call the this.triggerFunction?.stopTrigger() so that we actually terminate the trigger session from within lime-element

Reply:
Fixed it as it will come with a commit in another PR